hudi-utilities/src/main/java/org/apache/hudi/utilities/transform/ChainedTransformer.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.hudi.utilities.transform;
+
+import org.apache.hudi.common.util.TypedProperties;
+
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * An abstract {@link Transformer} to chain other {@link Transformer}s and apply sequentially.
+ * <p>
+ * A subclass is to supply a {@link List} of {@link Transformer}s in desired sequence.
+ */
+public abstract class ChainedTransformer implements Transformer {
```

Comment:
It's a user-faced feature, we'd better to describe it in the documentation. WDYT?

Reply:
Sure @yanghua . Could you point me to where I can add this?

Reply:
will do the docs change once this merges. thanks.